components/sensor/ld2420.rst

```diff
@@ -0,0 +1,261 @@
+  The recommended maximum value is ``12`` * ``0.75`` which results in approximately ``9m``.
+- **g#_move_threshold** (*Optional*, int): Gate threashold level for motion energy detection (Gate # from 0 to 15).
+  Above this level for the considered gate (distance), movement detection will be triggered.
+  Defaults to ``see table below``.
+- **g#_still_threshold** (*Optional*, int): Threshold for the Xth gate for still detection. (Gate # from 0 to 15).
+  Above this level for the considered gate (distance), still detection will be triggered.
+  Defaults to ``see table below``.
+
+.. list-table:: Default values for gate threshold
+    :widths: 25 25 25
+    :header-rows: 1
+
+    * - Gate:
+      - Default: Move threshold
+      - Default: Still threshold
+    * - 0
+      - 60000
+      - 40000
+    * - 1
+      - 30000
+      - 20000
+    * - 2
+      - 400
+      - 200
+    * - 3
+      - 300
+      - 250
+    * - 4
+      - 250
+      - 150
+    * - 5
+      - 250
+      - 150
+    * - 6
+      - 250
+      - 150
+    * - 7
+      - 250
+      - 150
+    * - 8
+      - 300
+      - 150
+    * - 9
+      - 250
+      - 150
+    * - 10
+      - 250
+      - 150
+    * - 11
+      - 250
+      - 150
+    * - 12
+      - 250
+      - 100
+    * - 13
+      - 200
+      - 100
+    * - 14
+      - 200
+      - 100
+    * - 15
+      - 200
+      - 100
+
+Sensor
+------
+
+The ``ld2420`` sensor allows you to use your :doc:`ld2420` to sense mmWave distance measurements.
+
+.. code-block:: yaml
+
+    sensor:
+      - platform: ld2420
+        moving_distance:
+          name : Moving Distance
+
+
+Configuration variables:
+************************
+
+- **moving_distance** (*Optional*): Distance of detected moving target.
+  All options from :ref:`Sensor <config-sensor>`.
+
+Binary Sensor
+-------------
+
+The ``ld2420`` binary sensor allows you to use your :doc:`ld2420` to sence presence.
+
+.. code-block:: yaml
+
+    binary_sensor:
+      - platform: ld2420
+        has_target:
+          name: Presence
+
+Configuration variables:
+************************
+
+- **has_target** (*Optional*): If true target detect either still or in movement.
+  All options from :ref:`Binary Sensor <config-binary_sensor>`.
+
+Important Information:
+**********************
+
+Solid objects and noise outside the ``detection_gate_max`` and ``detection_gate_min`` can cause false
+detections or abnormal gate thresholds. For example if the gate max range includes a wall it can potentially
+generate signal reflections. If you see unusual detections confirm the issue by placing the sensor in a
+completely open area. Never place two sensors in each others detection field, this will certainly
+cause false detections. Noise is a problem in many enviroments and future releases of this component
+will have an auto noise floor configuration option once Hi-Link provided full documentation.
```

Comment:
```suggestion
Solid objects and noise outside the ``detection_gate_max`` and ``detection_gate_min`` may cause false
detections or result in abnormal gate thresholds. For example, a wall within the gate max range can result
in signal reflections. If your sensor reports unexpected detections, you should test it by placing it in a
completely open room with no moving objects.

Also, never place two or more sensors in a manner such that their detection fields overlap, as this will certainly
result in false detections.

Noise is a problem in many environments and we anticipate that future releases of this component will have an auto
noise floor configuration option.
```